api/mesh/v1alpha1/gateway_route.proto

```diff
@@ -245,10 +245,21 @@ message MeshGatewayRoute {
         ];
       };
 
+      message Rewrite {
+        message ReplaceFull { string path = 1; }
+        message ReplacePrefixMatch { string path = 1; }
+
+        oneof path {
+          string replace_full = 1;
```

Comment:
I'm really struggling to understand how it works, shouldn't this be:
```
oneof path {
  ReplaceFull replace_full = 1;
  ReplacePrefixMatch replace_prefix_match = 2;
}
```
but at the same time generated code is correct